src/common/Tiles/MovieTilesContainer/MovieTile/index.js

```diff
@@ -18,26 +17,24 @@ import star from "./star.svg";
 
 export const MovieTile = ({ image, title, year, type, rate, votes }) => {
   return (
-    <TileButton>
```

Comment:
Got it! I knew that problem with this tiles was so small, but I can't deal with it by myself, SUPER, bravo!

Reply:
You're right. I haven't seen a problem with noImageMovie and noImagePerson for mobile displays. I'll try to solve it. Thanks